.gitignore

```diff
@@ -1,6 +1,9 @@
 /node_modules
 /public/hot
 /public/storage
+/public/css
```

Comment:
The file .gitignore is okay not to push

Reply:
The file .gitignore is okay not to push　？　so I just leave them? I don't know what your meaning

Reply:
@nontan0518 , The .gitignore is okay not to push. So just leave them in your local and then just add files if needed. But no need to push this file going to main branch

Reply:
thank you so much so just leave them